Trakt/Extensions.cs

```diff
@@ -146,29 +181,23 @@ public static bool MetadataIsDifferent(this TraktMovieCollected collectedMovie,
             return TraktResolution.uhd_4k;
         }
 
-        if (videoStream.Width.Value >= 1900)
+        if (videoStream.Width.Value >= 1400)
         {
             return videoStream.IsInterlaced ? TraktResolution.hd_1080i : TraktResolution.hd_1080p;
         }
 
-        if (videoStream.Width.Value >= 1270)
+        if (videoStream.Width.Value >= 950)
```

Comment:
looks very arbitrary?

Reply:
720p 4:4 is 960×720 and I gave it 10px leeway

Reply:
Moved them to consts and added some remarks to why they are as they are.